change/@azure-communication-react-61b4e975-0e4e-4d2f-a3c2-2eeb019b9b64.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "none",
```

Comment:
this is a prerelease change

Reply:
Since the spotlight feature is still in-progress, I thought any bug fixes/refactors should not be mentioned since spotlight has not made it to our release changelogs.

Reply:
Oh you're right! Sorry I had in my head spotlight was in beta already